### ISSUE TITLE
composite error with depth of field

### ISSUE THREAD

**Issue by Saiko125** [sixthsurge/photon#125]
### What happened?

composite2.fsh: composite2.fsh: 0(221) : error C1503: undefined variable "reverse_linear_depth"

### Minecraft Version

Other

### Which shader mod are you using?

Iris

### Shader mod version

1.6.14 (for minecraft 1.20.4

### Shader settings

```conf
#Fri Apr 05 07:14:02 CEST 2024
SKY_GROUND=true
COLORED_LIGHTS=true
AUTO_EXPOSURE=AUTO_EXPOSURE_SIMPLE
HANDHELD_LIGHTING=true
TAAU=true
BLOCK_ENTITY_SHADOWS=true
NORMAL_MAPPING=true
SH_SKYLIGHT=true
SNELLS_WINDOW=true
INFO=3
EDGE_HIGHLIGHT=true
MOTION_BLUR=true
POM=true
BLOCKY_CLOUDS=true
WATER_CAUSTICS=true
AIR_FOG_COLORED_LIGHT_SHAFTS=true
WATER_TEXTURE=WATER_TEXTURE_VANILLA
WATER_FOAM=true
shadowMapResolution=4096
SPECULAR_MAPPING=true
CLOUD_SHADOWS=true
DOF=true
VANILLA_SUN=true
```

**Comment by Saiko125** [comment]
[latest.log](https://github.com/sixthsurge/photon/files/14880152/latest.log)

here is output log I forgot to include

**Comment by sixthsurge** [comment]
fixed in cebd8278e9baa5313caa726eb8b447b286a054ea
